cni-plugin/integration/testutil/test_util.go

```diff
@@ -78,6 +143,10 @@ func (r *TestRunner) CheckCNIPluginIsLast() error {
 
 	plugins := conflist["plugins"].([]interface{})
 	lastPlugin := plugins[len(plugins)-1].(map[string]any)
+	err = checkLinkerdCniConf(lastPlugin)
+	if err != nil {
+		return err
+	}
```

Comment:
TIOLI:

```suggestion
	if err = checkLinkerdCniConf(lastPlugin); err != nil {
		return fmt.Errorf("Configuration contains erroneous value\n%s", err)
	}
```

I inlined the if statement (tbh I'm not the biggest fan of this but it makes sense here perhaps?) and then changed the error message to reflect where in the test the failure happened. By no means an improvement, just a suggestion, feel free to ignore both or either of them :)